src/main/java/github/weichware10/toolbox/AppController.java

```diff
@@ -0,0 +1,89 @@
+package github.weichware10.toolbox;
+
+import java.io.IOException;
+
+import github.weichware10.util.Logger;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'github.weichware10.util.Logger'